internal/cli/command/daemon/agent.go

```diff
@@ -75,41 +75,33 @@ func SetAgentConfigFromEnvs(cfg *config.Config) error {
 	if apiAddress != "" {
 		cfg.API.Address = apiAddress
 	}
-	apiTlsVerify := os.Getenv("LB_API_TLS_VERIFY")
-	if apiTlsVerify != "" {
-		cfg.API.TLS.Verify = converter.StringToBool(apiTlsVerify)
+	apiTLSVerify := os.Getenv("LB_API_TLS_VERIFY")
+	if apiTLSVerify != "" {
+		cfg.API.TLS.Verify = converter.StringToBool(apiTLSVerify)
 	}
-	apiTlsCa := os.Getenv("LB_API_TLS_CA_FILE")
-	if apiTlsCa != "" {
-		cfg.API.TLS.FileCA = apiTlsCa
+	apiTLSCa := os.Getenv("LB_API_TLS_CA_FILE")
+	if apiTLSCa != "" {
+		cfg.API.TLS.FileCA = apiTLSCa
 	}
-	apiTlsCert := os.Getenv("LB_API_TLS_CERT_FILE")
-	if apiTlsCa != "" {
-		cfg.API.TLS.FileCert = apiTlsCert
+	apiTLSCert := os.Getenv("LB_API_TLS_CERT_FILE")
+	if apiTLSCa != "" {
+		cfg.API.TLS.FileCert = apiTLSCert
 	}
-	apiTlsKey := os.Getenv("LB_API_TLS_PRIVATE_KEY_FILE")
-	if apiTlsCa != "" {
-		cfg.API.TLS.FileKey = apiTlsKey
+	apiTLSKey := os.Getenv("LB_API_TLS_PRIVATE_KEY_FILE")
+	if apiTLSCa != "" {
+		cfg.API.TLS.FileKey = apiTLSKey
 	}
-	workdir := os.Getenv("LB_WORKDIR")
-	if workdir != "" {
-		cfg.WorkDir = workdir
+	rootDir := os.Getenv("LB_ROOT_DIR")
+	if rootDir != "" {
+		cfg.RootDir = rootDir
 	}
-	manifefstdir := os.Getenv("LB_MANIFESTDIR")
-	if manifefstdir != "" {
-		cfg.ManifestDir = manifefstdir
+	storageDriver := os.Getenv("LB_STORAGE_DRIVER")
+	if rootDir != "" {
```

Comment:
Looks like you use incorrect variable. I think `storageDriver` should be instead of `rootDir`.